device_tags_python/requirements.txt

```diff
@@ -0,0 +1,8 @@
+click==8.0.4
+Flask==2.0.3
```

Comment:
why is the sdk installed directly on the docker file and everything else via the requirements.txt?
I would expect to have a dependency in this list for balena-sdk and nothing directly related to balena-sdk on the dockerfile. 

requirements.txt would be similar to your "package.json" or package-lock